src/features/home/ui/ChartHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import {
+  ChartHeaderContainer,
+  DateContainer,
+  HeaderDescription,
+  HeaderTitle,
+  IcArrow,
+  YearMonth,
+} from './chartHeader.style';
+import { IcArrowNext, IcArrowPrev } from '@/shared/assets/svg/index';
+
+interface Props {
+  title: string;
+  description: string;
+}
+
+const ChartHeader = ({ title, description }: Props) => {
+  const [yearMonthDate, setYearMonthDate] = useState(new Date());
+
+  const handleDate = (flag: number) => {
```

Comment:
우선 타입을 좁히는게 좋을 것 같은데요.
```suggestion
  const handleDate = (flag: -1 | 1) => {
```

flag라고 하는 매개변수를 받는 게 썩 좋지는 않은 것 같아요.
1, -1로 굳이 컨트롤을 한다면 매개변수 이름이 monthStep 이라던지 어떤 행위를 하기 위한 값인지 의도가 드러나야 할 것 같고 함수명 자체로 handleCalculateDate라던지 계산한다는 의도가 드러나야 할 것 같습니다.

방금 저도 이름을 지으면서 느꼈지만, 그리고 제가 제안해드린 이름도 썩 깔끔한 이름은 아닌 것 같다는 생각이 드는데요. 그 이유는 반대로 생각해보면 하나의 함수로 2가지의 케이스를 커버하고 있기 때문이라고 볼 수 있을 것 같아요. 그래서 저는 이전, 다음 클릭에 대한 각각의 함수를 만들어주는게 코드량이 늘어나더라도 훨씬 좋은 접근이라고 생각이 듭니다.

Reply:
추가로, **> 버튼**과 **< 버튼** 을 묶어서 생각하지 않고, 각각 단일의 기능을 하는 버튼으로 생각하면 이해가 편할 거 같아요! 
그렇게 되면 하나의 버튼에 하나의 기능을 수행할 수 있도록 설정할 수 있을 거 같습니다!

Reply:
말씀해주신대로 오히려 로직을 분리하니 훨씬 코드가 깔끔해졌습니다! 감사합니다!

```typescript
const handleAddMonth = (): void => {
    const newDate: Date = addMonths(yearMonthDate, 1);
    setYearMonthDate(newDate);
  };

  const handleSubMonth = (): void => {
    const newDate: Date = subMonths(yearMonthDate, 1);
    setYearMonthDate(newDate);
  };

  return (
    <ChartHeaderContainer>
      <div className="title-container">
        <HeaderTitle>{title}</HeaderTitle>
        <HeaderDescription>{description}</HeaderDescription>
      </div>
      <DateContainer>
        <IcArrow src={`${IcArrowPrev}`} alt="prev arrow icon" onClick={handleSubMonth} />
        <YearMonth>{`${format(yearMonthDate, 'yyyy년 MM월')}`}</YearMonth>
        <IcArrow src={`${IcArrowNext}`} alt="next arrow icon" onClick={handleAddMonth} />
      </DateContainer>
    </ChartHeaderContainer>
  );
```
https://github.com/GrebbMe/grabbme-fe/pull/89/commits/4113fda3217cb452417eb5d0abc5c6a3bac8120b 에서 반영했습니다!